Source/Scene/Scene.js

```diff
@@ -1695,6 +1729,7 @@ define([
 
         scene._commandList.length = 0;
         scene._overlayCommandList.length = 0;
+        scene._computeCommandList.length = 0;
```

Comment:
Picky, but probably put this before `_commandList` above since that is when it is executed.  In general, I would always keep this in the `compute command, draw/clear command, overlay command` order.